### DISCUSSION TITLE
Is such a scenario possible with this package?

### DISCUSSION THREAD

**Discussion by romatexdev** [def-studio/telegraph#668] [Q&A] [answered]
I am displaying a button for the user with parameters. 
`->keyboard(Keyboard::make()->row([
                    Button::make('Count')->action('count')->param('id', $order_id),
                ]))`

The user clicks on the button, the button method
displays an offer to write to the chat.
`public function count($id)
    {
        Telegraph::message('Change count')->send();
    }`

And then I need to intercept the entered text and the button parameter . How can I do this? I will be grateful for the sample code.

**Comment by romatexdev** [comment]
I searched through all the documentation and as far as I understood it is not feasible....Or am I missing something?

**Reply by fabio-ivona** [reply, marked as answer]
hi, you could use Telegraph's storage to keep track of the order ID and listen for messages in your webhook handler
